### PR TITLE
fix: Improve Handling of Sensitive Data in Browser

### PR DESCRIPTION
**Pull request by bhavanakarwade**
### What?
- Improved handling of sensitive data in browser.

### CODE REVIEW

**Review by pranalidhanavade — APPROVED** [review]
LG

**Review by Sheetal-ayanworks — APPROVED** [review]
LG

**Review by shitrerohit — APPROVED** [review]

File: src/app/LayoutCommon.astro [hunk 1]
```diff
@@ -3,14 +3,31 @@ import pkg from '../../package.json' assert { type: 'json' };
...
+
+const initData: Record<string, any> = {};
+const excludedEnvData: Record<string, any> = {};
+
+allEnvKeys.forEach((key) => {
+  const value = process.env[key] || import.meta.env[key];
+  
+  if (!value) return; 
+  
+  if (excludeKeys.includes(key)) {
```
Comment (ankita-p17) [inline src/app/LayoutCommon.astro:26]: This condition is not needed, as this const is never used.

**Review by ankita-p17 — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy-dev.yml [hunk 2]
```diff
@@ -22,8 +22,51 @@ jobs:
...
+
+         echo "PUBLIC_SHOW_NAME_AS_LOGO=true" >> .env
+
+         echo "PUBLIC_PLATFORM_NAME=CREDEBL" >> .env
+
+         echo "PUBLIC_PLATFORM_LOGO=/images/CREDEBL_ICON.png" >> .env
+
+         echo "PUBLIC_POWERED_BY=Blockster Labs Pvt. Ltd" >> .env
+
+         echo "PUBLIC_PLATFORM_DOCS_URL=https://docs.credebl.id/en/intro/what-is-credebl/" >> .env
```
[the remark this answers is not in the document]
Comment (bhavanakarwade) [reply .github/workflows/deploy-dev.yml:41]: I have added valid URL

File: src/app/LayoutCommon.astro (hunk 1 above)
Comment (bhavanakarwade) [reply src/app/LayoutCommon.astro]: Yes, I removed this condition, as we don't use this variable anywhere in the layout component.

**Review by ankita-p17 — APPROVED** [review]

**Pull request merged**